modules/accredited_representative_portal/app/services/accredited_representative_portal/power_of_attorney_request_service/accept.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module PowerOfAttorneyRequestService
+    class Accept
+      attr_reader :poa_request, :creator, :reason, :resolution
+      attr_accessor :form_submission
+
+      def initialize(poa_request, creator, reason)
+        @poa_request = poa_request
+        @creator = creator
+        @reason = reason
+      end
+
+      def call
+        ApplicationRecord.transaction do
+          create_resolution!
+          create_form_submission!
+        end
+
+        service = BenefitsClaims::Service.new(poa_request.claimant.icn)
+        response = service.submit2122(attributes)
```

Comment:
I think we could encapsulate all 3 of the methods here.

The line of thinking that led me here (and I think there are others) was that `attributes` isn't a good name, so then I thought of `service_payload` for accuracy against the current code structure, but then the redundancy of `service_` is potentially a smell (I really wouldn't go as far as calling it that though).

I don't think we necessarily need a class for it, but maybe a single entry function that might call helper functions with repetitively prefixed names (e.g. `submit` and `submit_payload`), just some thoughts.

The name `attributes` really does stick out though.